grifts.go

```diff
@@ -35,7 +36,7 @@ func secretGrift() {
 		rx := regexp.MustCompile(`(\W+)`)
 		bb = rx.ReplaceAll(bb, []byte(""))
 		s := randx.String(6) + string(bb)
-		fmt.Println(s[:127])
+		logrus.Infof(s[:127])
```

Comment:
This should be just a plain println so people can pipe the output easily into something else.